deploy/docs/additional_prometheus_configuration.md

```diff
@@ -2,7 +2,26 @@
 
 ## Configuration
 
-Prometheus configuration is located in `values.yaml` under `prometheus-operator` key for helm installation or in `prometheus-overrides.yaml` if you are using prometheus-operator directly. All changes described in this documentation should be introduced in those files depending of used deployment.
+Prometheus configuration is located in `values.yaml` under `prometheus-operator` key for helm installation.
+
+If the `prometheus-operator` has been installed directly, a `prometheus-overrides.yaml` should be generated
+using `docker`/`kubectl` and [sumologic-kubernetes-tools](prometheus-operator directly):
```

Comment:
> [sumologic-kubernetes-tools](prometheus-operator directly)

What do you want to link to?

Reply:
🤦

Reply:
https://github.com/sumologic/sumologic-kubernetes-tools#template-dependency-configuration